packages/expo-file-system/src/next/index.ts

```diff
@@ -0,0 +1 @@
+export { Directory, File, write } from './FileSystem';
```

Comment:
What would you say if we export these stuff as scoped like this:

```ts
import { Directory, File } from './FileSystem';

export const FileSystem = { Directory, File };
```

Reply:
I don't see a strong benefit, and it does make the import a bit more verbose or you then need to write `FileSystem.File` – it's kinda clear that the File, Directory imports are from filesystem, as long as there are only 2-3 of them I would not scope it.